### ISSUE TITLE
Token time out on Hub for large file upload

### ISSUE THREAD

**Issue by JimmyWhitaker** [pachyderm/pachyderm#5876]
Receiving the following error during a long upload from the local file system to a Pachyderm data repository. This was a new cluster, with a new token, and the upload had been running for a 48 hours successfully before throwing the following error. 

`error during authorization check for operation on \"bird_pics\": provided auth token is corrupted or has expired (try logging in again)`

**How to reproduce it (as minimally and precisely as possible)?**:
Begin a large upload from the local file system using `pachctl put` to Pachyderm Hub and monitor the output. 

**Anything else we need to know?**:

**Environment?**:
- Kubernetes version (use `kubectl version`):
Pachyderm Hub
- Pachyderm CLI version (use `pachctl version`):
Pachyderm Hub
```
$ pachctl version
COMPONENT           VERSION
pachctl             1.12.5
pachd               1.11.9
```
- Cloud provider (e.g. aws, azure, gke) or local deployment (e.g. minikube vs dockerized k8s):
- OS (e.g. from /etc/os-release):
```
NAME="Ubuntu"
VERSION="16.04.7 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.7 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```
- Others:

**Comment by jdoliner** [comment]
Are we only using a 48 hour expiration time for tokens in hub? That seems low to me.

**Comment by actgardner** [comment]
Hub tokens should last 30 days - it's hard-coded in the auth service, not even configurable. I'm assuming we no longer have access to the client so we can't run `pachctl auth whoami` to see if the token is permanently expired or if this was transient?

**Comment by nitinjainsj** [comment]
@JimmyWhitaker to answer question above

**Comment by JimmyWhitaker** [comment]
> Hub tokens should last 30 days - it's hard-coded in the auth service, not even configurable. I'm assuming we no longer have access to the client so we can't run `pachctl auth whoami` to see if the token is permanently expired or if this was transient?

No, we don't have access to that client anymore (it was a temporary AWS machine).

**Comment by mindthevirt** [comment]
<div>Agent Dale Georg linked Freshdesk ticket <a href="https://pachyderm-support.freshdesk.com/a/tickets/281">281</a> for this issue.</div>
